frontend/components/LineCountList.tsx

```diff
@@ -22,6 +22,7 @@ export default function LineCountList() {
       <div className={"messagelist-header"}> Current Line Count : </div>
       {fetchedLines.map((line, index) => (
         <LineCount
+          style={{ fontSize: 10 }}
```

Comment:
@AIndoria maye have something to say about inline styles

Reply:
removed, not necessary as I was only testing and forget to delete

Reply:
...I was about to go on a rant about how you'd be throwing me in the ditch if you used inline styles since I'm mostly focusing on css files.